pkg/maps/ctmap/ctmap.go

```diff
@@ -215,8 +215,15 @@ type GCFilter struct {
 
 	// MatchIPs is the list of IPs to remove from the conntrack table
 	MatchIPs map[string]struct{}
+
+	// MatchCB is called, when non-nil, if filtering by ValidIPs and MatchIPs
+	// passes. Returning true keeps the CT entry, and false deletes it.
+	MatchCB MatchCBFunc
```

Comment:
nit:
```suggestion
	DeleteEntry DeleteEntryCB
```

or

```suggestion
	KeepEntry KeepEntryCB
```

(given that the boolean result of this function has an impact on whether connections are kept in the connection tracker or not)

Or alternatively, keep it as match and just don't return the boolean for deletion handling - I don't think that functionality is actually used anywhere right? So unless we have a plan for it to ever return false, why even support that for now?

Reply:
I felt bad adding it and calling it matchCB without it doing something "legit". I'll switch it to purely be there to emit the entry and drop the boolean return part. That will be more clear.